pkgs/os-specific/linux/rtl88x2bu/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, kernel, bc }:
+
+stdenv.mkDerivation rec {
+  name = "rtl88x2bu-git-${kernel.version}-${version}";
```

didn't notice this earlier, git checkout is denoted by "unstable-YYYY-MM-DD"
```suggestion
  name = "rtl88x2bu-${kernel.version}-${version}";
```